pkg/capabilities/triggers/mercury_trigger.go

```diff
@@ -49,6 +49,7 @@ type MercuryTriggerService struct {
 	stopCh             services.StopChan
 	wg                 sync.WaitGroup
 	lggr               logger.Logger
+	Meta               datastreams.SignersMetadata // usually empty, but set to a value in mock trigger
```

Comment:
To make is slightly cleaner, let's call it metaOverride (private) and set via NewMercuryTriggerService().

Reply:
I added a separate setter method for it so existing code still compiles with this change